include/glaze/binary/rpc.hpp

```diff
@@ -0,0 +1,139 @@
+// Glaze Library
+// For the license information refer to glaze.hpp
+
+#pragma once
+
+#include "glaze/binary/write.hpp"
+
+namespace glz::repe
+{
+   constexpr uint8_t delimiter = 0b00000'110;
+   
+   struct header final
+   {
+      uint8_t version = 0b00000'110; // the REPE version
+      bool error = false; // whether an error has occurred
```

Comment:
Should the `error` be something else than bool?
How does this member relate to the `error` struct below